frontend/services/viewer/viewerService/unityController.ts

```diff
@@ -0,0 +1,511 @@
+/**
+ **  Copyright (C) 2020 3D Repo Ltd
+ **
+ **  This program is free software= you can redistribute it and/or modify
+ **  it under the terms of the GNU Affero General Public License as
+ **  published by the Free Software Foundation, either version 3 of the
+ **  License, or (at your option) any later version.
+ **
+ **  This program is distributed in the hope that it will be useful,
+ **  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ **  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ **  GNU Affero General Public License for more details.
+ **
+ **  You should have received a copy of the GNU Affero General Public License
+ **  along with this program.  If not, see <http=//www.gnu.org/licenses/>.
+ **/
+
+import EventEmitter from 'eventemitter3';
+import { VIEWER_EVENTS } from '../../../constants/viewer';
+import { DialogActions } from '../../../modules/dialog';
+import { dispatch } from '../../../modules/store';
+import { PIN_COLORS } from '../../../styles';
+
+declare const UnityUtil;
+
+export class UnityController {
```

Comment:
I dont understand the reason for the existence of this file as well as the viewercontroller.

Reply:
The reason for creating unityController class was a need to separate the methods responsible only for controlling Unity, from the remaining actual logic part.

Of course, the old classes should be removed, I have it noted in the description https://github.com/3drepo/3drepo.io/issues/1916

Nevertheless, I am open to ideas about the names of these classes as I did not have good ideas for them.

Reply:
Yeah, well, if you see the methods in this file is almost a 1:1 relation to unity util,  it doesnt add much, it just feels like a proxy to unityutil.

Reply:
Agree with @sanmont3drepo 

1. I'm not sure why ViewerService is renamed to ViewerController?
2. I think the inheritance doesn't make much sense, instead of calling super, you may as well just call UnityUtils directly. If the idea is that it allows for future implementations of viewers with different rendering engine then the inheritance seems to be done wrong. ViewerController should generic parent class that unityController extends. But I think this is overkill. Just merging the 2 is fine.